public/redux/reducers/ad.ts

```diff
@@ -492,10 +492,8 @@ export const startHistoricalDetector = (
   startTime: number,
   endTime: number
 ): APIAction => {
-  const baseUrl = `${AD_NODE_API.DETECTOR}/${detectorId}`;
-  const url = dataSourceId
-    ? `${baseUrl}/${dataSourceId}/start`
-    : `${baseUrl}/start`;
+  const baseUrl = `${AD_NODE_API.DETECTOR}/${detectorId}/start`;
```

Comment:
should we have `_start` instead of `start`? or both can be used?
ref: https://opensearch.org/docs/latest/observing-your-data/ad/api/#start-detector-job

Reply:
we have to use `start` here. It's different from the `_start` in the doc you pasted